serving/src/main/java/ai/djl/serving/util/ClusterConfig.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.serving.util;
+
+import ai.djl.util.Utils;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+/** A class that holds cluster configurations. */
+public final class ClusterConfig {
+
+    private static final ClusterConfig INSTANCE = new ClusterConfig();
+
+    private int clusterSize;
+    private CountDownLatch latch;
+    private String error;
+
+    private ClusterConfig() {
+        clusterSize = Integer.parseInt(Utils.getenv("DJL_CLUSTER_SIZE", "1"));
```

Comment:
It might be slightly nicer to configure this using the config manager rather than just environment variables

Reply:
1. This is configured by LWS, and it only support env var so far
2. There are other code (PyEnv) is also using just env var